src/app/components/Account/AccountLink.tsx

```diff
@@ -7,6 +7,8 @@ import { RouteUtils } from '../../utils/route-utils'
 import Typography from '@mui/material/Typography'
 import { COLORS } from '../../../styles/theme/colors'
 import { SearchScope } from '../../../types/searchScope'
+import { isValidEthAddress } from '../../utils/helpers'
+import { toChecksumAddress } from '@ethereumjs/util'
 
 export const AccountLink: FC<{
```

Comment:
Copy To Clipboard still copies non-checksum address.
And I don't think CopyToClipboard component should be doing transformations, so the best place to convert is in useGetRuntimeEvents, but it's also the hardest; event body is `any` and different event types have different fields.

Reply:
> Copy To Clipboard still copies non-checksum address.

Indeed. Do we want it something else? I assumed we want the checksum format for displaying only.

> I don't think CopyToClipboard component should be doing transformations,

Nor do I, that's why I added it to the `AccountLink` component.

>  so the best place to convert is in useGetRuntimeEvents,

I'm not sure about that. We are using `AccountLink` in many other places, too, where it's not related to runtime event. To me, `AccountLink` seems to be the natural choice.

> but it's also the hardest; event body is any and different event types have different fields.

When formatting runtime events, when we know that the type is address, we always use `AccountLink` for displaying, so adding it here seems to solve the `any` problem as well as we currently can.

Reply:
I think it should copy checksum format too. API returns checksum format everywhere else, so display and copy of e.g. addresses in Transaction Details are both in checksum format. Same in list of Tokens.

Reply:
>  the best place to convert is in useGetRuntimeEvents

OK, moved tra transformation into `useGetRuntimeEvents`.